files/en-us/web/css/anchor-size/index.md

```diff
@@ -156,82 +178,75 @@ We specify three {{htmlelement("div")}} elements, one `anchor` element and the t
 
 ```css hidden
 .anchor {
   font-size: 2rem;
   color: white;
   text-shadow: 1px 1px 1px black;
   background-color: hsl(240 100% 75%);
-  width: 100px;
-  height: 100px;
   text-align: center;
-  line-height: 100px;
-  border-radius: 10px;
-  border: 1px solid black;
-  padding: 3px;
+  align-content: center;
+  outline: 1px solid black;
 }
 
 body {
   width: 80%;
   margin: 0 auto;
 }
-```
-
-```css
-.anchor {
-  anchor-name: --myAnchor;
-}
-```
 
-The positioned elements are then associated with the anchor element by setting its anchor name as the value of the positioned elements' `position-anchor` properties. We also set the positioned elements' {{cssxref("position")}} properties to `fixed`, so they can be positioned relative to the anchor's position on the page.
-
-```css hidden
 .infobox {
+  align-content: center;
   color: darkblue;
   background-color: azure;
-  border: 1px solid #ddd;
-  padding: 10px;
-  border-radius: 10px;
+  outline: 1px solid #ddd;
   font-size: 1rem;
   text-align: center;
 }
 ```
 
 ```css
+.anchor {
+  anchor-name: --myAnchor;
+  width: 100px;
+  height: 100px;
+}
+
 .infobox {
   position-anchor: --myAnchor;
   position: fixed;
 }
 ```
 
-Finally, we set some distinct property values on the positioned elements:
+We set some distinct property values on the positioned elements:
 
-- Different {{cssxref("inset-area")}} values position the elements in different places around the anchor element.
-- The {{cssxref("width")}} of the first infobox is set to double the anchor element's width. `anchor-size(width)` retrieves the anchor element width, and this is multiplied by two inside a {{cssxref("calc()")}} function. The {{cssxref("height")}} of the second infobox is set to two-thirds of the anchor element's height, using a similar technique.
-- Appropriate margin values are set on the positioned elements to give them some separation from the anchor element.
+- The positioned elements are tethered to the anchor with different {{cssxref("inset-area")}} values that position the elements in different places around the anchor element.
+- The {{cssxref("height")}} of the first infobox is set to the height of the anchor height. `anchor-size(height)` returns the anchor element's width. Using the `anchor-size()` function within a {{cssxref("calc()")}} function, the {{cssxref("width")}} is set to double the anchor element's width; `anchor-size(width)` retrieves the anchor element's width, which is then multiplied by two.
+- The {{cssxref("height")}} of the second infobox is set to two-thirds of the anchor element's height, using a similar technique.
+- Margin values are included to provide some separation from the anchor element.
 
 ```css
 #infobox1 {
   inset-area: right;
+  height: anchor-size(height);
   width: calc(anchor-size(width) * 2);
   margin-left: 5px;
 }
 
 #infobox2 {
-  inset-area: top span-left;
+  inset-area: top span-right;
```

Comment:
it was going off the left, and that's not what we're showing, and i didn't want to explain wrapping, so moved to right.